### PR TITLE
deps: bump runc/libcontainer 1.0.3 -> 1.1.2

### PR DESCRIPTION
**Pull request by kolyshkin**
#### What type of PR is this?

/kind dependency-change

#### What this PR does / why we need it:

Bump runc from 1.0.3 to 1.1.2. For release notes, see https://github.com/opencontainers/runc/releases

Modify internal/config/cgmgr/cgroupfs.go to use a new (better, simpler)
way of getting a cgroup manager.

#### Which issue(s) this PR fixes:

None

#### Does this PR introduce a user-facing change?

```release-note
Update runc/libcontainer dependency to v1.1.2
```

### CODE REVIEW

File: internal/config/cgmgr/cgroupfs.go [hunk 1]
```diff
@@ -167,15 +160,24 @@ func setWorkloadSettings(cgPath string, resources *rspec.LinuxResources) error {
...
 	}
 
+	var paths map[string]string
 	if node.CgroupIsV2() {
-		var err error
-		mgr, err = fs2.NewManager(cg, cgPath, rootless.IsRootless())
-		if err != nil {
-			return err
-		}
+		paths := make(map[string]string, 1)
```
Comment (haircommander) [inline internal/config/cgmgr/cgroupfs.go:165]: this is a newly scoped variable, not the paths defined above

**Comment by haircommander** [comment]
one nit, otherwise LGTM

**Comment by haircommander** [comment]
also this needs an update to dependencies.yaml https://github.com/cri-o/cri-o/runs/4526579891?check_suite_focus=true

File: internal/config/cgmgr/cgroupfs.go (hunk 1 above)
Comment (kolyshkin) [reply internal/config/cgmgr/cgroupfs.go:165]: oopsie )

**Comment by adrianreber** [comment]
Would it also make sense to update `scripts/versions` to use the latest runc in CI. (Just asking because I need some of the 1.1.0 features).

**Comment by kolyshkin** [comment]
> Would it also make sense to update `scripts/versions` to use the latest runc in CI. (Just asking because I need some of the 1.1.0 features).

I think I did that already

**Comment by haircommander** [comment]
/hold 

openshift tests haven't passed in a day here

**Comment by kolyshkin** [comment]
Rebased; moved to runc 1.1.0.

**Comment by kolyshkin** [comment]
Some of the errors are caused by network glitches:


```
the build rpms failed after 3m26s with reason DockerBuildFailed: Dockerfile build strategy has failed.

http://vault.centos.org/centos/7/os/Source/repodata/repomd...:1f13:2e7:2901:d2b0:c3b2:9347:934c: Network is unreachable"
http://vault.centos.org/centos/7/os/Source/repodata/repomd...:1f13:2e7:2901:d2b0:c3b2:9347:934c: Network is unreachable"
http://vault.centos.org/centos/7/os/Source/repodata/repomd...:1f13:2e7:2901:d2b0:c3b2:9347:934c: Network is unreachable"
[ERROR] hack/build-rpms.sh:31: `yum-builddep -y "${OS_RPM_SPECFILE}"` exited with status 1.
```

**Comment by kolyshkin** [comment]
rebased on top of main HEAD

**Review by saschagrunert — APPROVED** [review]
/retest-required
LGTM

**Comment by TomSweeneyRedHat** [comment]
LGTM
once tests are happy

**Comment by kolyshkin** [comment]
Better late than never, let's revive this one.

**Comment by kolyshkin** [comment]
Rebased to resolve a merge conflict caused by #5906.

**Comment by saschagrunert** [comment]
Whoops sorry for that @kolyshkin 🙏

**Pull request closed without merging**